### PR TITLE
feat: integrate avatarapi to pre-fill avatar

### LINKED ISSUE #13121: [CAL-2886] implement avatarapi.com to pre-fill avatars
when making an account, we should prefill the avatar based on the email thats used to signup

CleanShot 2024-01-09 at 15 24 18@2x

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2886](https://linear.app/calcom/issue/CAL-2886/implement-avatarapicom-to-pre-fill-avatars)</sub>

### PR DESCRIPTION
**Pull request by Amit91848**
## What does this PR do?

Fixes #13121 

https://github.com/calcom/cal.com/assets/74371312/d3651621-436b-409e-b206-d00a6a9f17e5

## Requirement/Documentation

[Avatar Api Docs](https://docs.avatarapi.com/#usage)

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

Create account on [Avatar Api](https://www.avatarapi.com/)
Add Username and Password in .env

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by PeerRich** [comment]
awesome!

File: .env.example [hunk 1]
```diff
@@ -138,6 +138,11 @@ FORMBRICKS_HOST_URL=https://app.formbricks.com
 FORMBRICKS_ENVIRONMENT_ID=
 FORMBRICKS_FEEDBACK_SURVEY_ID=
 
+# AvatarAPI
+# Used to pre-fill avatar during signup
+AVATARAPI_USERNAME=
+AVATARAPI_PASSWORD=
```
Comment (PeerRich) [inline .env.example:144]: added to vercel ✅

**Review by PeerRich — COMMENTED** [review]
nice!

**Review by PeerRich — APPROVED** [review]
tested. works like a charm:

CleanShot 2024-01-18 at 22 36 21



waiting for code quality review

**Review by emrysal — APPROVED** [review]
LGTM,  code looks good

**Review by zomars — APPROVED** [review]

**Pull request merged**